src/gnatss/configs/main.py

```diff
@@ -118,12 +119,16 @@ class Configuration(BaseConfiguration):
     site_id: str = Field(..., description="The site identification for processing")
     # TODO: Separate settings out to core plugin
     solver: Optional[Solver] = Field(None, description="Solver configurations")
-    output: OutputPath
+    posfilter: Optional[PositionFilter] = Field(
+        None, description="Position filter configurations"
+    )
+    output: Optional[OutputPath] = Field(None, description="Output path configurations")
 
     def __init__(self, **data):
         super().__init__(**data)
 
-        # Set the transponders pxp id based on the site id
-        transponders = self.solver.transponders
-        for idx in range(len(transponders)):
-            transponders[idx].pxp_id = "-".join([self.site_id, str(idx + 1)])
+        if self.solver is not None:
+            # Set the transponders pxp id based on the site id
+            transponders = self.solver.transponders
+            for idx in range(len(transponders)):
+                transponders[idx].pxp_id = "-".join([self.site_id, str(idx + 1)])
```

Comment:
This is fixing a simple bug... before this assumes that the `Solver` config exists, but it's not always.